server/plugin.go

```diff
@@ -367,6 +367,43 @@ func (p *Plugin) GetToDo(ctx context.Context, username string, githubClient *git
 	return text, nil
 }
 
+func (p *Plugin) HasNews(info *GitHubUserInfo) bool {
+	username := info.GitHubUsername
+	ctx := context.Background()
+	githubClient := p.githubConnect(*info.Token)
+	config := p.getConfiguration()
+
+	issues, _, err := githubClient.Search.Issues(ctx, getReviewSearchQuery(username, config.GitHubOrg), &github.SearchOptions{})
+	if err != nil {
+		mlog.Error(err.Error())
+		return false
+	}
+
+	notifications, _, err := githubClient.Activity.ListNotifications(ctx, &github.NotificationListOptions{})
```

Comment:
We actually don't consider all the GitHub notifications as notifications in the plugin since some of them are super spammy. So you'll need to a bit of post-processing here to filter out the ones we don't care about. See https://github.com/mattermost/mattermost-plugin-github/blob/master/server/plugin.go#L298 for what we skip/ignore

Reply:
Thanks! I will read up on that.